Samples/EventHorizon/Server2/Bootloader.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading.Tasks;
+using Dolittle.Booting;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Dolittle.Runtime.Server
```

Comment:
Same as the other server. Leverage the existing Runtime.Server project as a reference and call its bootloader.